index.js

```diff
@@ -374,6 +398,125 @@ const getSecurityVulnerabilitiesForGitHubRepo = async (
   return allVulnerabilities
 }
 
+/**
+ * Get any vulnerability related to the specified Docker image using a NodeJS
+ * version that has reached End-Of-Life (EOL).
+ *
+ * @param dockerImage
+ * @param versionsCsvUrl
+ * @returns {Promise<undefined | {severity, summary, detailsUrl, ecosystem, identifiers, inspectedOn, packageName, vulnerableVersionRange}>}
+ */
+const getNodeJsVersionEolVulnerability = async (dockerImage, versionsCsvUrl) => {
+  const currentDate = getHyphenatedDate()
+  const nodeJsVersion = await docker.getNodeJsVersionOfImageUsingCache(dockerImage, versionsCsvUrl)
+  if (nodeJsVersion === 'NONE') {
+    console.log(`(NodeJS is not used in ${dockerImage})`)
+    return undefined
+  } else if (nodeJsVersion === 'UNKNOWN') {
+    const unknownDockerImageWarning = `No NodeJS version data found in spreadsheet for ${dockerImage}`
+    console.warn(`WARNING: ${unknownDockerImageWarning}`)
+    return {
```

Comment:
Could you make a local variable with the return value, and only modify the properties that are different in the three different cases?

Reply:
I could, yeah. It would be intermixing the data returned for these three different cases in exchange for removing a handful of lines of code. Do you think that would be worth it? (I'm leaning towards not, but could probably be convinced otherwise.)

Reply:
Your call, since it's fine as it is, but I see the advantage being more than just removing a few lines. It should make it easier to see the differences in each case.